crates/analyzer/tests/snapshots/errors__bad_visibility.snap

```diff
@@ -3,11 +3,47 @@ source: crates/analyzer/tests/errors.rs
 expression: error_string_ingot(&path)
 
 ---
-error: unresolved path item
+error: type MyInt is private
+  ┌─ compile_errors/bad_visibility/src/main.fe:1:11
+  │
+1 │ use foo::{MyInt, MY_CONST, MyStruct, MyTrait, my_func, MyContract, MyEnum }
+  │           ^^^^^ MyInt
+
+error: constant MY_CONST is private
+  ┌─ compile_errors/bad_visibility/src/main.fe:1:18
+  │
+1 │ use foo::{MyInt, MY_CONST, MyStruct, MyTrait, my_func, MyContract, MyEnum }
+  │                  ^^^^^^^^ MY_CONST
+
+error: struct MyStruct is private
+  ┌─ compile_errors/bad_visibility/src/main.fe:1:28
+  │
+1 │ use foo::{MyInt, MY_CONST, MyStruct, MyTrait, my_func, MyContract, MyEnum }
+  │                            ^^^^^^^^ MyStruct
+
+error: trait MyTrait is private
+  ┌─ compile_errors/bad_visibility/src/main.fe:1:38
+  │
+1 │ use foo::{MyInt, MY_CONST, MyStruct, MyTrait, my_func, MyContract, MyEnum }
+  │                                      ^^^^^^^ MyTrait
+
+error: function my_func is private
+  ┌─ compile_errors/bad_visibility/src/main.fe:1:47
+  │
+1 │ use foo::{MyInt, MY_CONST, MyStruct, MyTrait, my_func, MyContract, MyEnum }
+  │                                               ^^^^^^^ my_func
+
+error: type MyContract is private
+  ┌─ compile_errors/bad_visibility/src/main.fe:1:56
+  │
+1 │ use foo::{MyInt, MY_CONST, MyStruct, MyTrait, my_func, MyContract, MyEnum }
+  │                                                        ^^^^^^^^^^ MyContract
+
+error: type MyEnum is private
   ┌─ compile_errors/bad_visibility/src/main.fe:1:68
   │
 1 │ use foo::{MyInt, MY_CONST, MyStruct, MyTrait, my_func, MyContract, MyEnum }
-  │                                                                    ^^^^^^ not found
```

Comment:
Not strictly related to the fix but the test was wrong in that it imported a non existing Enum rather than an enum that isn't `pub`.